packages/website/docs/DocSearch-program.md

```diff
@@ -0,0 +1,68 @@
+---
+title: DocSearch program
+---
+
+If you're not finding the answer to your question on this website, this page will help you. If you're still unsure, don't hesitate to send [your question to us][1] directly.
+
+For questions related to the DocSearch x Algolia Crawler, please see our [Crawler FAQ](/docs/crawler).
+
+## What do I need to install on my side?
+
+Nothing.
```

Comment:
I understand the point but it's also wrong since you need a frontend :p

Reply:
True :D Stated the frontend implementation